### PR TITLE
feat: print files artifacts registered in an enclave during enclave inspect

### LINKED ISSUE #176: [FR]: `kurtosis files ls`
### Background and motivation

I'm building the quickstart, and I wanted to see if my `upload_files` worked (though I assuemd it did because the plan worked ;))

### Current behavior

<no option>

### Desired behavior

`kurtosis files ls` shows the files artifacts in an enclave

### How important is this feature or improvement to you?

Painful, the lack of this feature makes using Kurtosis frictionful.

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
Closes https://github.com/kurtosis-tech/kurtosis/issues/176

### CODE REVIEW

**Comment by h4ck3rk3y** [comment]
image

File: cli/cli/commands/enclave/inspect/files_artifacts.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while fetching enclave with name '%v'", enclaveInfo.GetName())
+	}
+
+	filesArtifactsNamesAndUuids, err := enclaveContext.GetAllFilesArtifactNamesAndUuids(ctx)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while fetching files artifacts name and uuids for enclave '%v'", enclaveContext.GetEnclaveName())
+	}
+
+	sort.Slice(filesArtifactsNamesAndUuids, func(i, j int) bool {
```
Comment (Peeeekay) [inline cli/cli/commands/enclave/inspect/files_artifacts.go:30]: nit; maybe add a comment to what its's doing.

File: core/server/commons/enclave_data_directory/files_artifact_store.go [hunk 2]
```diff
@@ -145,6 +145,21 @@ func (store FilesArtifactStore) ListFiles() map[string]bool {
 	return artifactNameSet
 }
 
+func (store FilesArtifactStore) GetFileNamesAndUuids() []FileNameAndUuid {
```
Comment (Peeeekay) [inline core/server/commons/enclave_data_directory/files_artifact_store.go:148]: nit: maybe add like a simple unit test for this method.

File: cli/cli/commands/enclave/inspect/files_artifacts.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	"github.com/kurtosis-tech/stacktrace"
+	"sort"
+)
+
+const (
+	fileUuidsHeader = "UUID"
+	fileNameHeader  = "Name"
+)
+
+func printFilesArtifacts(ctx context.Context, kurtosisCtx *kurtosis_context.KurtosisContext, _ backend_interface.KurtosisBackend, enclaveInfo *kurtosis_engine_rpc_api_bindings.EnclaveInfo, showFullUuids bool, _ bool) error {
```
Comment (Peeeekay) [inline cli/cli/commands/enclave/inspect/files_artifacts.go:20]: nit: could you add unit test for this one.

File: cli/cli/commands/enclave/inspect/files_artifacts.go (hunk 1 above)
Comment (Peeeekay) [inline cli/cli/commands/enclave/inspect/files_artifacts.go]: nit: maybe add a comment on we are sorting file-artifacts on name in asc order.

**Review by Peeeekay — APPROVED** [review]

**Review by Peeeekay — APPROVED** [review]

**Pull request merged**